README.md

```diff
@@ -1 +1,173 @@
-# group-capstone
\ No newline at end of file
+# group-capstone
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ group-capstone] 
+
+> This project is the second capstone project from Microverse. we created webapp by following all the requirements needed and choose our favorite  Api , tvmaze API to get info about movies, and 'involvment' API to Send & get likes and comments.
+
+**[ group-capstone]** 
+
+## 🛠 Built With 
+HTML/CSS
+JavaScript(ES6)
+Webpack ✈️
+tvmaze API and Involvment API
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://ibtisam34.github.io/group-capstone/dist/)
+
+- > a link to our Kanban board  https://github.com/Ibtisam34/kanban-board/projects/1
+- >Approved issue by microverse reviewers https://github.com/Ibtisam34/kanban-board/issues/21
+> @ibtisam34 proudest reviewed pull-request https://github.com/Ibtisam34/group-capstone/pull/3
+> 
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+$ git clone https://github.com/Ibtisam34/group-capstone.git $ 
+cd group-capstone
+$ npm i
+$npm start 
+
+### Prerequisites
+
+In order to run this project you need:
+$ npm start
+
+### Setup
+
+Clone this repository to your desired folder:
+$ git clone https://github.com/Ibtisam34/group-capstone.git
+
+
+### Install
+
+Install this project with:
+$npm i
+
+### Usage
+
+To run the project, execute the following command:
+
+$npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+#npm test
+
+### Deployment
+
+You can deploy this project using:
+
+  npm run dev
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Riyana**
+
+- GitHub: [@ibtisam34](https://github.com/ibtisam34)
+- Twitter: [@Queenjin2](https://twitter.com/Queenjin2)
+
+
+👤 **Joseph David**
+
+- GitHub: [@jodavid444](https://github.com/jodavid444)
+- Twitter: [@jodavid444dave](https://twitter.com/jodavid444dave)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/joseph-david-01a8a5243)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
```

Comment:
- [ ] [OPTIONAL] Let's add some of the future features you'd like to implement in the future.